mne/evoked.py

```diff
@@ -455,10 +455,10 @@ def plot_joint(self, times="peaks", title='', picks=None,
 
     @fill_doc
     def animate_topomap(self, ch_type=None, times=None, frame_rate=None,
-                        vmin=None, vmax=None, butterfly=False, blit=True,
-                        show=True, time_unit='s', sphere=None, *,
-                        image_interp=_INTERPOLATION_DEFAULT,
-                        extrapolate=_EXTRAPOLATE_DEFAULT, verbose=None):
+                        butterfly=False, blit=True, show=True, time_unit='s',
+                        sphere=None, *, image_interp=_INTERPOLATION_DEFAULT,
+                        extrapolate=_EXTRAPOLATE_DEFAULT, verbose=None,
+                        vmin=None, vmax=None):
```

Comment:
here too, move `vmin` and `vmax` to come before `verbose`

Reply:
Thanks for your guidance (and patience!)